LoopKit/DeliveryLimits.swift

```diff
@@ -44,12 +44,12 @@ public extension DeliveryLimits.Setting {
         }
     }
     
-    var descriptiveText: String {
+    func descriptiveText(appName: String) -> String {
```

Comment:
Is it common that descriptiveText is assumed to be localized? If not, consider adding `localized` to the func name to make that clear.